### ISSUE TITLE
Improve accessibility of student view

### ISSUE THREAD

**Issue by payten** [sakaiproject/sakai#3361]
We've spent much of our energy improving the accessibility of the grade spreadsheet, but the student grade summary table could also use some attention.

For example, we should ensure the table has a caption and all rows and headers are marked up correctly.  Also, labels, aria-label or aria-describedby could be used on certain content to provide better context.

**Comment by kyleblythe** [comment]
Hi @payten ,

Found one little text edit for these captions:

+studentsummary.categorytoggle = Show / hide the gradenook items within this category

Should be:

+studentsummary.categorytoggle = Show / hide the gradebook items within this category

That's it, though! The descriptions all look great.

**Comment by payten** [comment]
Thanks @kyleblythe!  The Gradenook, sponsored by Barnes and Noble.

**Comment by kyleblythe** [comment]
Looks good!
